package.json

```diff
@@ -56,12 +53,15 @@
     "markdown-loader": "0.1.7",
     "marked": "0.3.5",
     "moment": "2.10.6",
+    "ntypescript": "1.201602032217.1",
```

Comment:
actually, looks opposite - you used `tsc` in `prepublish` which is said to be deprecated (in here https://github.com/basarat/tsc) and linked to `ntypescript` as a suggestion to move onto.